.changeset/many-maps-lay.md

```diff
@@ -0,0 +1,7 @@
+---
+'@openfn/language-salesforce': minor
+---
+
+- Add `insert()` function which is an alias for `create()`.
+- Improve jsdocs for query, bulk, describe, create, upsert
+- Deprecated `upsertIf()` and `createIf()`
```

Comment:
this note should show an example of usage with fnIf

Reply:
Updated the changeset